src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts

```diff
@@ -62,7 +59,6 @@ export class CodebasesItemActionsComponent implements OnDestroy, OnInit {
             this.workspaceBusy = false;
             if (workspaceLinks != undefined) {
               let name = this.getWorkspaceName(workspaceLinks.links.open);
-              this.windowService.open(workspaceLinks.links.open, name);
```

Comment:
How does it open if you remove this line? Am I missing it someplace else?

Reply:
It opens when the open button is hit on the `codebases-item-workspaces` component [0]

https://github.com/aptmac/fabric8-ui/blob/c2e016d124b0a13174d21419c44864a209ec2266/src/app/space/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.ts#L126

Reply:
so what did the open you remove do? open it a 2nd time?

Reply:
No, this open triggers immediately after workspace creation, which causes it to hit the ad-blocker if present.

This modified behaviour mirrors what was changed in the `codebases-item-workspaces` component in [[1]](https://github.com/fabric8-ui/fabric8-ui/pull/2972), in which the creation and open logic were decoupled.

[1] https://github.com/fabric8-ui/fabric8-ui/pull/2972

Reply:
ok